spec/lib/email_alert_signup_api_spec.rb

```diff
@@ -13,103 +13,78 @@
       applied_filters: applied_filters,
       default_filters: default_filters,
       facets: facets,
-      subscriber_list_title: subscriber_list_title,
+      subscriber_list_title: "title",
     )
   end
 
   let(:default_filters) { {} }
   let(:applied_filters) { {} }
   let(:facets) { [] }
-  let(:subscriber_list_title) { "Subscriber list title" }
-  let(:slug) { "slug" }
-  let(:subscription_url) { "/email/subscriptions/new?topic_id=#{slug}" }
-
-  def init_simple_email_alert_api(slug)
-    stub_email_alert_api_has_subscriber_list(
-      "tags" => {},
-      "slug" => slug,
-    )
-  end
 
-  describe "default_attributes" do
+  describe "#signup_url" do
     context "no default_attributes or attributes" do
-      describe "#signup_url" do
-        it "returns the url email-alert-api gives back" do
-          req = init_simple_email_alert_api(slug)
+      it "returns the url email-alert-api gives back" do
+        req = stub_email_alert_api_has_subscriber_list(
+          "tags" => {},
+          "slug" => "slug",
+        )
 
-          expect(subject.signup_url).to eql subscription_url
-          assert_requested(req)
-        end
+        subscription_url = "/email/subscriptions/new?topic_id=slug"
+        expect(subject.signup_url).to eql subscription_url
```

Comment:
does `signup_api_wrapper` == `subject`?

Reply:
Ooh, good spot - I've added another commit to remove those inconsistencies. Do you want to check it over?

Reply:
Looks good!